contributing.md

```diff
@@ -1,14 +1,175 @@
+VM](https://cloud.centos.org/centos/8/x86_64/images/CentOS-8-GenericCloud-8.1.1911-20200113.3.x86_64.qcow2)
+and execute the command and download the package
+`standard-test-roles-inventory-qemu` from the Fedora repository:
+
+```dnf install standard-test-roles-inventory-qemu```
+
+Note that the last path is the one of the test you want to run:
+
+`TEST_SUBJECTS=CentOS-8-GenericCloud-8.1.1911-20200113.3.x86_64.qcow2
+ansible-playbook -v -i /usr/share/ansible/inventory/standard-inventory-qcow2
+tests/test_default.yml`
+
+4. Once the work is ready and commited, push the branch to your remote fork and click on
+   "new Pull Request" on Github.
+
+5. All set! Now wait for the continuous integration to pass and go over your commit if
+   there are any errors. If there is no problem with your contribution, the mantainer
+   will merge it to the main project.
+
+### Some important tips
+
+- Make sure your fork and branch are up-to-date with the main project. First of all,
+  [configure a remote upstream for your
+fork](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/configuring-a-remote-for-a-fork),
+and keep your branch up-to-date with the upstream using ```git pull --rebase upstream
+master```
+
+- Try to make a commit per issue.
+
+- If you are asked to make changes to your PR, don't panic! Many times it is enough to
+  amend your previous commit adding the new content to it (`git commit --amend`). Be
+sure to pull the latest upstream changes after that, and use `git push
+--force-with-lease` to re-upload your commit with the changes!  Another way of doing
+changes to a PR is by [squashing
+commits](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-request-merges#squash-and-merge-your-pull-request-commits).
+
+- There are times when someone has made changes on a file you were modifying while you
+  were making changes to your unfinished commit. At times like this, you need to make a
+[**rebase**](https://help.github.com/en/github/using-git/about-git-rebase) with
+conflicts. On the rebase you have to compare what the other person added to what you
+added, and merge both file versions into one that combines it all.
+
+- If you have any doubt, do not hesitate to ask! You can join IRC channel \#systemroles
+  on freenode, or ask on the PR/issue itself.
+
+### Naming Ansible Items
+
+* All YAML or Python files, variables, arguments, repositories, and other such names
+  should follow standard Python naming conventions of being in
+  `snake_case_naming_schemes`.
+
+* Names should be mnemonic and descriptive and not strive to shorten more than
+  necessary. Systems support long identifier names, so use them to be descriptive
+
+* All defaults and all arguments to a role should have a name that begins with the role
+  name to help avoid collision with other names. Avoid names like `packages` in favor of
+  a name like `foo_packages`. 
```

Comment:
```suggestion
  a name like `network_packages`. 
```

Since this is for the network role, we can use a specific example.

Reply:
Fixed this too